### PR TITLE
Feature/user profile

### LINKED ISSUE #18: User Profile WireFrame
Wireframe for a family's user profile

### PR DESCRIPTION
**Pull request by abodnar1**
## Summary:
Add skeleton structure for UserProfile component with hard coded data. Once the queries and data is retrieved, we can pass the user info as props.

## Type of Changes:
- [ ] Bug fix
- [ ] Refactor
- [X] New feature

## How was this tested:
Chrome Dev Tools

## What are the relevant tickets:
* close #18 
* relevant #13 , #14 , #15 , #16

### CODE REVIEW

File: src/Components/UserProfile/UserProfile.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import "./UserProfile.css";
+
+// Does this component need state?
+// How does the data look? Will be grabbing a user via ID
+
+// Will need to map over user tags
+// Will need to map over the events
+// Will need a different view for current logged in user vs. other user
+
```
Comment (tfoxcollis) [inline src/Components/UserProfile/UserProfile.js:10]: Love the psuedocoding list!

**Review by tfoxcollis — COMMENTED** [review]
Looks good Amber!

File: public/index.html [hunk 2]
```diff
@@ -10,6 +10,8 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+
```
Comment (BlueNealis) [inline public/index.html:14]: ooooh you addded a font :o

File: src/Components/UserProfile/UserProfile.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        <p className="description-text-box">
+          Description of your family
+        </p>
+      
+        <div className="tag-container">
+          <p className="tag-title">2 Kids</p>
+          <p className="tag-title">MLM</p>
+          <p className="tag-title">Married</p>
+          <p className="tag-title">Monogamous</p>
+        </div>
```
Comment (BlueNealis) [inline src/Components/UserProfile/UserProfile.js:38]: Excited to make maps for these little tags here <3

**Review by BlueNealis — APPROVED** [review]
I love it~

**Pull request merged**